Makefile.w32 cannot build: does not create kcolor.obj.

I tried installing gifsicle on Windows 11 by running `nmake.exe -f Makefile.w32` in the src directory and got the following errors:
`gifwrite.obj : error LNK2019: unresolved external symbol _gamma_tables referenced in function _gfc_init`
`quantize.obj : error LNK2001: unresolved external symbol _gamma_tables`
and so on, I included the full error list at the bottom.

I was able to fix this by adding "kcolor.obj" to _GIFSICLE\_OBJS_ on line 27:
`GIFSICLE_OBJS = clp.obj fmalloc.obj giffunc.obj gifread.obj gifunopt.obj \`
`	$(GIFWRITE_OBJ) merge.obj optimize.obj quantize.obj support.obj \`
`	xform.obj kcolor.obj gifsicle.obj`
as well as inserting this below line 54:
`kcolor.obj: ..\config.h gifsicle.h kcolor.c`

It's been a few years since I did anything in C so while this seems to fix the issue, I really don't know what I'm doing/if this is a proper solution. 

Full error paste:
`gifwrite.obj : error LNK2019: unresolved external symbol _gamma_tables referenced in function _gfc_init`
`quantize.obj : error LNK2001: unresolved external symbol _gamma_tables`
`xform.obj : error LNK2001: unresolved external symbol _gamma_tables`
`optimize.obj : error LNK2019: unresolved external symbol _kchist_init referenced in function _initialize_optimizer`
`xform.obj : error LNK2001: unresolved external symbol _kchist_init`
`optimize.obj : error LNK2019: unresolved external symbol _kchist_cleanup referenced in function _initialize_optimizer`
`xform.obj : error LNK2001: unresolved external symbol _kchist_cleanup`
`gifsicle.obj : error LNK2001: unresolved external symbol _kchist_cleanup`
`optimize.obj : error LNK2019: unresolved external symbol _kchist_add referenced in function _all_colormap_add`
`xform.obj : error LNK2001: unresolved external symbol _kchist_add`
`quantize.obj : error LNK2019: unresolved external symbol _kc_revgamma_transform referenced in function _colormap_diversity`
`xform.obj : error LNK2001: unresolved external symbol _kc_revgamma_transform`
`quantize.obj : error LNK2019: unresolved external symbol _kc_oklab_transform referenced in function _colormap_stream`
`quantize.obj : error LNK2019: unresolved external symbol _kd3_cleanup referenced in function _colormap_stream`
`xform.obj : error LNK2001: unresolved external symbol _kd3_cleanup`
`quantize.obj : error LNK2019: unresolved external symbol _kd3_build_xradius referenced in function _colormap_image_atkinson`
`quantize.obj : error LNK2019: unresolved external symbol _kd3_init_build referenced in function _colormap_stream`
`xform.obj : error LNK2001: unresolved external symbol _kd3_init_build`
`quantize.obj : error LNK2019: unresolved external symbol _kd3_closest_transformed referenced in function _colormap_image_atkinson`
`xform.obj : error LNK2001: unresolved external symbol _kd3_closest_transformed`
`xform.obj : error LNK2019: unresolved external symbol _kd3_init referenced in function _scale_image_prepare`
`xform.obj : error LNK2019: unresolved external symbol _kd3_add_transformed referenced in function _kd3_add8g`
`xform.obj : error LNK2019: unresolved external symbol _kd3_build referenced in function _scale_image_prepare`
`xform.obj : error LNK2019: unresolved external symbol _kchist_compress referenced in function _scale_image_add_colors`
`gifsicle.obj : error LNK2019: unresolved external symbol _kc_set_gamma referenced in function _merge_and_write_frames`
`gifsicle.obj : error LNK2019: unresolved external symbol _kchist_make referenced in function _do_colormap_change`
`gifsicle.exe : fatal error LNK1120: 16 unresolved externals`
`NMAKE : fatal error U1077: 'cl -I.. -I..\include -DHAVE_CONFIG_H -D_CONSOLE /W2 /O2 /Fegifsicle.exe clp.obj fmalloc.obj giffunc.obj gifread.obj gifunopt.obj  gifwrite.obj merge.obj optimize.obj quantize.obj support.obj  xform.obj gifsicle.obj /link setargv.obj' : return code '0x2'`

--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,8 @@ stamp-h*
 ungifsicle-*.rpm
 ungifsicle-*.tar.gz
 compile
+*.obj
+*.exe
+*.pdb
+*.ilk
+*.idb

--- a/src/Makefile.w32
+++ b/src/Makefile.w32
@@ -24,9 +24,9 @@ LDLIBS = setargv.obj
 
 GIFSICLE_OBJS = clp.obj fmalloc.obj giffunc.obj gifread.obj gifunopt.obj \
 	$(GIFWRITE_OBJ) merge.obj optimize.obj quantize.obj support.obj \
-	xform.obj gifsicle.obj
+	xform.obj gifsicle.obj kcolor.obj
 
-GIFDIFF_OBJS = clp.obj fmalloc.obj giffunc.obj gifread.obj gifdiff.obj
+GIFDIFF_OBJS = clp.obj fmalloc.obj giffunc.obj gifread.obj gifdiff.obj kcolor.obj
 
 .c.obj:
 	$(CC) $(CFLAGS) /c $<

